hocuspocus-server/src/setMark.ts

```diff
@@ -23,7 +24,75 @@ export function setMark(
 ) {
   // needed to get the pmSchema
   // if you use a BlockNote custom schema, make sure to pass it to the create options
-  const editor = ServerBlockNoteEditor.create();
+  const editor = ServerBlockNoteEditor.create({
+    _extensions: {
+      // TODO to get this to work, I needed to pass a copy of the comment mark to get it into the schema
+      comments: Mark.create({
+        name: "comment",
+        excludes: "",
+        inclusive: false,
+        keepOnSplit: true,
+        group: "blocknoteIgnore", // ignore in blocknote json
```

Comment:
We need to figure out what to do about this...

Reply:
good catch! this was a leftover, it's been fixed already. will make a PR